Core/GDCore/Project/Layer.cpp

```diff
@@ -29,6 +29,11 @@ void Layer::SetCameraCount(std::size_t n) {
 void Layer::SerializeTo(SerializerElement& element) const {
   element.SetAttribute("name", GetName());
   element.SetAttribute("visibility", GetVisibility());
+  element.SetAttribute("lightingLayer", GetLightingLayer());
+  element.SetAttribute("syncWithBaseLayer", GetSyncWithBaseLayer());
+  element.SetAttribute("r", (int)GetAmbientLightColorRed());

```

Comment:
Prefer saving attributes with the same names as their variables: `ambientLightColorR`, same for G and B